<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLMainWidget</class>
 <widget class="QWidget" name="TLMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TLMainWidget</string>
  </property>
  <widget class="TLAnimationButton" name="m_homeButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="TLSwitchButton" name="m_languageSwitchButton" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="TLAnimationButton" name="m_whiteBoardButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>160</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_connectionPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>connection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_userLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_userLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>userName:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_leftPushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_ipLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ip:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_portLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_portLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="IpWidget" name="m_ipWidget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TLSwitchButton</class>
   <extends>QWidget</extends>
   <header>TLSwitchButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TLAnimationButton</class>
   <extends>QPushButton</extends>
   <header>TLAnimationButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IpWidget</class>
   <extends>QWidget</extends>
   <header>IpWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
